backend/src/types/urlSafety.test.ts

```
@@ -0,0 +1,177 @@
+    }
+  });
+
+  it('refuses a path that names an authority other than ours', () => {
+    for (const value of FOREIGN_AUTHORITY_PATHS) {
+      expect(editAccepts({ imageUrl: value }), value).toBe(false);
+    }
+  });
+
+  it('accepts the absolute http(s) url every stored picture actually is', () => {
+    expect(editAccepts({ imageUrl: 'https://upload.wikimedia.org/x.jpg' })).toBe(true);
+    expect(editAccepts({ imageUrl: 'http://whc.unesco.org/document/1' })).toBe(true);
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)